pkg/utils/pod/provisioning.go

```diff
@@ -12,22 +12,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package scheduling
+package pod
 
 import (
+	"github.com/awslabs/karpenter/pkg/controllers/provisioning/v1alpha1"
 	"go.uber.org/zap"
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-)
-
-var (
-	IgnoredOwners []schema.GroupVersionKind = []schema.GroupVersionKind{
-		{Group: "apps", Version: "v1", Kind: "DaemonSet"},
-	}
 )
 
 func IsNotIgnored(pod *v1.Pod) bool {
-	for _, ignoredOwner := range IgnoredOwners {
+	for _, ignoredOwner := range v1alpha1.IgnoredOwners {
```

Comment:
Careful -- this dependency arrow will very quickly get you into circular dependencies. Utils should be though of as a leaf of a tree. Either put IgnoredOwners into this file, or pass it in as a parameter. IMO, it's fine if scheduler.IsNotIgnored knows about daemonsets (i.e., ignored owners lives in this file), since daemonsets are ignored by the scheduler.

Reply:
I'll pass it into this file. I think it fits here.